modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -0,0 +1,36 @@
+= Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can connect apps to Anypoint Platform without granting them unrestricted access to data or having to share sensitive credentials.
+When you grant an app access, you have direct control over its data access, usage, and credentials.
+You can audit the use of data by connected apps, and you can also revoke access at any time.
+
+== Authentication
+
+Use OAuth 2.0 and OpenID to authorize apps to access resources using tokens or Single Sign-On (SSO).
+Additionally, you can share account data with third-party applications.
```

Comment:
Why is sharing data _additionally_ to accessing resources? your data is one of the resources that can be accessed